app/src/main/kotlin/com/simplemobiletools/clock/extensions/Context.kt

```diff
@@ -144,11 +143,23 @@ fun Context.setupAlarmClock(alarm: Alarm, triggerInSeconds: Int) {
     val targetMS = System.currentTimeMillis() + triggerInSeconds * 1000
     try {
         AlarmManagerCompat.setAlarmClock(alarmManager, targetMS, getOpenAlarmTabIntent(), getAlarmIntent(alarm))
+
+        // Trigger a notification to dismiss the alarm 5 minutes before the alarm if the screen is on
+        val dismissalTriggerTime = if (targetMS - System.currentTimeMillis() < (5.minutes.inWholeMilliseconds))(System.currentTimeMillis() + 500) else targetMS - (5.minutes.inWholeMilliseconds)
```

Comment:
theres way too many things on this line, simplify it please

Reply:
done